github/pulls_comments.go

```diff
@@ -13,19 +13,20 @@ import (
 
 // PullRequestComment represents a comment left on a pull request.
 type PullRequestComment struct {
-	ID               *int64     `json:"id,omitempty"`
-	InReplyTo        *int64     `json:"in_reply_to,omitempty"`
-	Body             *string    `json:"body,omitempty"`
-	Path             *string    `json:"path,omitempty"`
-	DiffHunk         *string    `json:"diff_hunk,omitempty"`
-	Position         *int       `json:"position,omitempty"`
-	OriginalPosition *int       `json:"original_position,omitempty"`
-	CommitID         *string    `json:"commit_id,omitempty"`
-	OriginalCommitID *string    `json:"original_commit_id,omitempty"`
-	User             *User      `json:"user,omitempty"`
-	Reactions        *Reactions `json:"reactions,omitempty"`
-	CreatedAt        *time.Time `json:"created_at,omitempty"`
-	UpdatedAt        *time.Time `json:"updated_at,omitempty"`
+	ID                  *int64     `json:"id,omitempty"`
+	InReplyTo           *int64     `json:"in_reply_to,omitempty"`
+	Body                *string    `json:"body,omitempty"`
+	Path                *string    `json:"path,omitempty"`
+	DiffHunk            *string    `json:"diff_hunk,omitempty"`
+	PullRequestReviewID *int       `json:"pull_request_review_id,omitempty"`
```

Comment:
Let's please make this ID an `int64` (not an `int`) to be consistent with the recent ID changes.